src/modules/shared/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { Sidebar, Menu, MenuItem } from "react-pro-sidebar";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+export default function SideBar() {
+  const [isCollapse, setIsCollapse] = useState(true);
```

Comment:
We can save this state in local storage and retrieve it when reloading.

Reply:
done hope you review the new code